packages/cli/src/tools/config/resolveNodeModuleDir.ts

```diff
@@ -1,15 +1,2 @@
-import path from 'path';
-
-/**
- * Finds a path inside `node_modules`
- */
-export default function resolveNodeModuleDir(
-  root: string,
-  packageName: string,
-): string {
-  return path.dirname(
-    require.resolve(path.join(packageName, 'package.json'), {
-      paths: [root],
-    }),
-  );
-}
+import {resolveNodeModuleDir} from '@react-native-community/cli-tools';
```

Comment:
I kept this module in place to keep the mocking model intact. I initially tried removing it and reworking the mocks, but the global mock of `cli-tools` made that impossible.